bin/kubectl-rabbitmq

```diff
@@ -286,13 +290,21 @@ main() {
         ;;
     "install-cluster-operator")
         shift 1
-        if [[ "$#" -gt 1 ]]; then
+        if [[ "$#" -gt 0 ]]; then
             usage
             exit 1
         fi
-        install_cluster_operator "${1:-}"
+        install_cluster_operator
+        ;;
```

Comment:
I think this was used to install the Operator using a custom/relocated image. Are we removing support for that feature?

Reply:
This feature was removed in https://github.com/rabbitmq/cluster-operator/commit/d5444f53103d3bba91050390b907c6222338eb32. If we want that feature, we could add it back after this PR has been merged?

Reply:
@Zerpet, I created https://github.com/rabbitmq/cluster-operator/issues/446 which would require this flag. So, we can add it back when working on that issue.

Reply:
Ah! I wasn't aware of d5444f53103d3bba91050390b907c6222338eb32, my bad 🙂 I'm happy to conside re-adding this feature as part of #446.